### ISSUE TITLE
Error running the daemon

### ISSUE THREAD

**Issue by gellesrinivas** [loki-project/loki-gui#11]
Getting error while running the daemon. When I look at the log. I see below

2018-05-10 14:36:46.453  13488  ERROR  net.http  src/wallet/node_rpc_proxy.cpp:98  Failed to connect to daemon
2018-05-10 14:38:41.276  13488  ERROR  wallet.wallet2  src/wallet/wallet2.cpp:4008

**Comment by Doy-lee** [comment]
Hi, can I get information on what OS you're running on and what you've tried? 
#12 has suggested builds for mac, and the releases page has updated Linux builds that work on more distributions.

**Comment by Doy-lee** [comment]
Have you tried launching the daemon separately first and letting the GUI connect to it? Similarly, please ensure lokid.exe is in the same directory as loki-wallet-gui.exe and has not been quarantined by any antivirus software.

**Comment by gellesrinivas** [comment]
Yes I did and that worked. The instructions were not clear as what to do first. I am good now. Thanks

Gelle
